src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -522,7 +502,8 @@ const EditHomepage = ({ match }) => {
 
       switch (elemType) {
         case "section": {
-          const val = enumSection(value, false)
+          // NOTE: setting `isError` to true to get an empty object
+          const val = enumSection(value, true)
           const err = enumSection(value, true)
```

Comment:
Hmm if we're always retrieving an empty object now, do we even need `enumSection` to take in the boolean `isError`

I'm fine with leaving it if it takes a lot of effort to remove though

Reply:
idts but i lazy to remove HAHA

Reply:
removed